content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -96,6 +96,32 @@ gateways is [actively in development](/docs/setup/upgrade/gateways/) and is cons
     $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
     {{< /text >}}
 
+### Stable revision labels (experimental)
+
+{{< warning >}}
+This workflow is only supported from Istio versions 1.10 and above.
+{{< /warning >}}
```

Comment:
This is only going into the 1.11 docs.  Are you implying "to upgrade from versions 1.10 and above" ?

Reply:
This is supported in Istio 1.10 as well. The hope was to get this merged into istio.io before we cut for 1.11, but that doesn't appear likely now. Users of 1.10 can do these steps as it's not a new feature.

Reply:
I'm confused by this statement. It seems to imply that you must be upgrading `from` Istio 1.10 and above. You've used the boilerplates and show here an upgrade from 1.9 to 1.10. Maybe you mean `to` 1.10 and then from 1.9 or 1.8?

Reply:
I changed this to hopefully make it clear what helm supports going forward.